tests/unittest/_torch/sampler/test_beam_search.py

```
@@ -702,81 +802,95 @@ def test_finish_beams():
-
-    request.set_generated_tokens(
-        torch.randint(0,
-                      vocab_size, (beam_width, num_generated_tokens),
-                      dtype=torch.int32).tolist())
+    @contextmanager
+    def _uut_provider(
+            is_warmup: bool) -> Generator[Callable[[], None], None, None]:
+        test_params = GeneralTestParams()
+        beam_width = test_params.beam_width
+        num_generated_tokens = test_params.num_generated_tokens
+        test_params.seq_len
```

Ruff (0.15.0)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)